src/zalopay/__tests__/__mocks__/refund/responses.ts

```diff
@@ -0,0 +1,14 @@
+export const refundCreateSuccess = {
+  "return_code": 1,
+  "return_message": "Giao dịch thành công",
+  "sub_return_code": 1,
+  "sub_return_message": "Giao dịch thành công",
+  "refund_id": 123456789,
+};
+
+export const refundQuerySuccess = {
```

Comment:
this response looks weird to me, why we don't have any data on it?

Reply:
This is the real data response when I tested on SB env

Reply:
@leonthaibao I thinks we can copy example response from our docs: https://docs.zalopay.vn/downloads/api/ZaloPay-APIs-Integration-Document.pdf.  
It'll be more consistent.